site/test/test.sh

```diff
@@ -132,9 +132,9 @@ test_redirect "$TEST_BASE_URL/download/plugins/lolwut/latest/git.hpi" "https://u
 # Ensure that ?uctest gets a tiny OK response
 OUTPUT="$( curl -I "$TEST_BASE_URL/update-center.json?uctest" 2>/dev/null )"
 assert 'HTTP/1.1 200 OK'   "$( echo "$OUTPUT" | dos2unix | grep -F 'HTTP/1.1' )"
-assert 'Content-Length: 3' "$( echo "$OUTPUT" | dos2unix | grep -F 'Content-Length: ' )"
+assert 'Content-Length: 3' "$( echo "$OUTPUT" | dos2unix | grep -Fi 'Content-Length: ' )"
```

Comment:
I don't think this works. Confirm by testing locally with
```suggestion
assert 'content-length: 3' "$( echo "$OUTPUT" | dos2unix | grep -Fi 'Content-Length: ' )"
```

Reply:
🤔 weird I tested it and it worked. But I see my mistake: the assertion tests for exact matching so this PR is incomplete. Putting back to draft.

Reply:
FT, I got the following (with `bash -x ./test.sh`):

```
++ dos2unix
++ grep -Fi 'Content-Length: '
+ assert 'Content-Length: 3' 'Content-Length: 3'
+ local 'expected=Content-Length: 3'
+ local 'actual=Content-Length: 3'
+ [[ Content-Length: 3 != \C\o\n\t\e\n\t\-\L\e\n\g\t\h\:\ \3 ]]
++ curl -I 'http://localhost:8080/foo/update-center.json?uctest'
+ OUTPUT='HTTP/1.1 200 OK
Date: Thu, 17 Oct 2024 16:39:49 GMT
Server: Apache/2.4.62 (Unix)
Last-Modified: Thu, 17 Oct 2024 15:23:06 GMT
ETag: "3-624adc36a073c"
Accept-Ranges: bytes
Content-Length: 3
Content-Type: application/json
'
++ echo 'HTTP/1.1 200 OK
Date: Thu, 17 Oct 2024 16:39:49 GMT
Server: Apache/2.4.62 (Unix)
Last-Modified: Thu, 17 Oct 2024 15:23:06 GMT
ETag: "3-624adc36a073c"
Accept-Ranges: bytes
Content-Length: 3
Content-Type: application/json
'
++ dos2unix
++ grep -F HTTP/1.1
+ assert 'HTTP/1.1 200 OK' 'HTTP/1.1 200 OK'
+ local 'expected=HTTP/1.1 200 OK'
+ local 'actual=HTTP/1.1 200 OK'
+ [[ HTTP/1.1 200 OK != \H\T\T\P\/\1\.\1\ \2\0\0\ \O\K ]]
++ echo 'HTTP/1.1 200 OK
Date: Thu, 17 Oct 2024 16:39:49 GMT
Server: Apache/2.4.62 (Unix)
Last-Modified: Thu, 17 Oct 2024 15:23:06 GMT
ETag: "3-624adc36a073c"
Accept-Ranges: bytes
Content-Length: 3
Content-Type: application/json
'
++ dos2unix
++ grep -Fi 'Content-Length: '
+ assert 'Content-Length: 3' 'Content-Length: 3'
+ local 'expected=Content-Length: 3'
+ local 'actual=Content-Length: 3'
+ [[ Content-Length: 3 != \C\o\n\t\e\n\t\-\L\e\n\g\t\h\:\ \3 ]]
+ cleanup
+ docker stop update-center2-test
update-center2-test
```

=> in my case, the local container answers Camel cased answered so it works. but it would not pass on updates.jenkins.io of course